pxd.c

```diff
@@ -1536,17 +1659,15 @@ static void pxd_abort_context(struct work_struct *work)
 
 	BUG_ON(fc->connected);
 
-	printk(KERN_INFO "PXD_TIMEOUT (%s:%u): Aborting all requests...",
+	printk(KERN_ERR "PXD_TIMEOUT (%s:%u): Aborting all requests...",
 		ctx->name, ctx->id);
 
 	fc->allow_disconnected = 0;
 
 	/* Let other threads see the value of allow_disconnected. */
 	synchronize_rcu();
 
-	spin_lock(&fc->lock);
 	fuse_end_queued_requests(fc);
-	spin_unlock(&fc->lock);
```

Comment:
This is the rootcause of https://jenkins.portworx.dev/job/PX2-RELEASE/view/PX2-RELEASE-Cloudsnap/job/FVT-MN-ObjectStore-NEXT-PX2-RELEASE/